apps/entity-service/src/entities/dto/site.dto.ts

```diff
@@ -67,10 +71,13 @@ export class SiteLightDto extends EntityDto {
 }
 
 export type AdditionalSiteFullProps = AdditionalProps<SiteFullDto, SiteLightDto, Omit<Site, "project">>;
+export type AdditionalSiteLightProps = Pick<SiteFullDto, "treesPlantedCount">;
+
+export type AdditionalSiteCombinedProps = AdditionalSiteFullProps & AdditionalSiteLightProps;
```

Comment:
The full props should always directly extend from the light props because the full props must always be an explicit super set of light props. So I'd do this instead as 
```
export type AdditionalSiteFullProps = AdditionalSiteLightProps & AdditionalProps<SiteFullDto, SiteLightDto, Omit<Site, "project">>;
```
and remove this combined props type.